parlai/scripts/multiprocessing_train.py

```diff
@@ -84,7 +84,11 @@ def multiprocess_train(
         distributed_utils.sync_object(None)
 
         # Run the actual training
-        return single_train.TrainLoop(opt).train()
+        retval = single_train.TrainLoop(opt).train()
+
+        dist.destroy_process_group()
```

Comment:
huh, okay. That makes sense. Should you drop the one in the tearDown in the test?

Reply:
yes, this actually was causing an issue where any subsequent test called after `test_singletask_distributed` would have `is_distributed()` return True, since `dist.is_initialized()` would be True. What do you mean by dropping the tearDown in the test?